### DISCUSSION TITLE
Terminal colors

### DISCUSSION THREAD

**Discussion by dantonyuk** [pwntester/octo.nvim#218] [Q&A]
Are you planning to support ctermfg/ctermbg colors in highlight groups?

**Comment by pwntester** [comment]
Do you means adding them to the default highlight groups? 

Im not planning on adding them in the near future, but Im ok adding them if you send a PR for it

**Reply by dantonyuk** [reply]
I believe it's enough to mention in the documentation that terminal users should enable `termguicolors`.

vim:
```vim
set termguicolors
```

Lua:
```
vim.opt.termguicolors = true
```

**Reply by pwntester** [reply]
Can you please update the docs and send a PR? Thanks!
